README.md

```diff
@@ -762,21 +762,33 @@ MODELS=`[
     {
        "name": "gemini-1.5-pro",
        "displayName": "Vertex Gemini Pro 1.5",
+       "multimodal": true,
        "endpoints" : [{
           "type": "vertex",
           "project": "abc-xyz",
           "location": "europe-west3",
-          "model": "gemini-1.5-pro-preview-0409", // model-name
+          "model": {
+            "id": "some-id",
+            "name": "gemini-1.5-pro-preview-0409"
+          },
 
           // Optional
           "safetyThreshold": "BLOCK_MEDIUM_AND_ABOVE",
           "apiEndpoint": "", // alternative api endpoint url,
-          // Optional
           "tools": [{
```

Comment:
grounding is not supported with multimodal yet.
So tools or multimodal are mutualy exclusive